addon/components/image/index.js

```diff
@@ -0,0 +1,43 @@
+import Component from '@ember/component';
```

Comment:
`import Component from '@glimmer/component';`

Reply:
That wont be an option for now. We would need to rewrite all components to support `this.args` :/